curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md

```diff
@@ -17,6 +17,8 @@ Start this project on Repl.it using <a href='https://repl.it/github/freeCodeCamp
 
 <section id='instructions'>
 
+**HINT:** Do not forget to use a body parsing middleware to handle the post requests. Also, you can use the function `dns.lookup(host, cb)` from the `dns` core module to verify a submitted URL. You can learn about this function from its [documentation](https://nodejs.org/api/dns.html)
```

Comment:
```suggestion
**HINT:** Do not forget to use a body parsing middleware to handle the POST requests. Also, you can use the function `dns.lookup(host, cb)` from the `dns` core module to verify a submitted URL. You can learn about this function from its [documentation](https://nodejs.org/api/dns.html).
```
Just a slight nitpick. Otherwise, LGTM.

Reply:
Really would like to not have the link to the Node documentation.

Reply:
@RandellDawson Shall I remove the whole sentence, or just the link?

Reply:
We don't have any lessons on `dns` do we? 🤔

Reply:
No, but the user can look it up in the Node documentation if they want to.

Reply:
> Shall I remove the whole sentence, or just the link?

I was thinking the whole sentence should be removed.